test/Microsoft.AspNetCore.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -146,6 +145,11 @@ public static TheoryData<BindingInfo> NullAndEmptyBindingInfo
             }
         }
 
+        // Make sure the metadata is honored when a [ModelBinder] attribute is associated with an action parameter's
+        // type. This should behave identically to such an attribute on an action parameter. (Tests such as
```

Comment:
`This should behave identically to such an attribute` => `This should behave identically to an attribute`